coderd/coderdtest/coderdtest.go

```diff
@@ -531,7 +531,7 @@ func NewProvisionerDaemon(t testing.TB, coderAPI *coderd.API) io.Closer {
 	}()
 
 	daemon := provisionerd.New(func(ctx context.Context) (provisionerdproto.DRPCProvisionerDaemonClient, error) {
-		return coderAPI.CreateInMemoryProvisionerDaemon(ctx, t.Name())
+		return coderAPI.CreateInMemoryProvisionerDaemon(ctx, "test")
```

Comment:
Is this change caused by validation constraints?

Reply:
yep I didn't think of that constraint at the time, and `t.Name()` seemed like a reasonable thing to use.